packages/kolibri-common/composables/useBaseSearch.js

```diff
@@ -278,6 +279,13 @@ export default function useBaseSearch({
       if (get(isUserLoggedIn)) {
         fetchContentNodeProgress?.(getParams);
       }
+
+      const params = get(route).params;
+      if (params?.quizId) {
+        getParams.contains_quiz = true;
```

Comment:
This is not always true, we should set this filter to true just for practice quizzes, for example this is a non-practice quiz that appears while browsing the folder tree:

![image](https://github.com/user-attachments/assets/cf02dbf4-d740-47b9-a39d-1687be868ad1)

If I search by its name, It wont appear:

![image](https://github.com/user-attachments/assets/ec1aa212-baf0-433d-95c5-7745b1e76f8e)

This doesnt happen in lessons (i.e. its not a problem with the keywords filter):
![image](https://github.com/user-attachments/assets/70a8898a-2612-48c4-9e93-a0f004e3cc18)